.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "Rust"]
+	path = Rust
```

Comment:
Being nit-picky, but can you make it lower-case. This keeps the "languages" all the same and the named implementations are Capitalized.

Happy to be disagreed with.

Reply:
Sure, I've changed it to lower-case.